editor/src/main/java/net/rpgtoolkit/editor/editors/TileSetCanvas.java

```diff
@@ -52,7 +52,7 @@ public TileSetCanvas(TileSet tileset) {
 
     this.tileset = tileset;
     int width = 320;

```

Comment:
Hmm the whole thing is fixed at the moment to 320px wide, and the only supported tile size technically is 32x32, for now anyway. 

RPGMaker actually has the same restriction where you can't stretch the tileset palette across the screen, as it has a fixed width of 10. This will all need changing if other sizes are to be supported like 16x16 (common SNES tile size). I'm going to leave it for now though XD